pkgs/data/fonts/open-sans/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitLab }:
+
+stdenv.mkDerivation rec {
+  pname = "open-sans";
+  version = "1.11";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "fonts-team";
+    repo = "fonts-open-sans";
+    rev = "fa55fea78526fb0356be9cdd8f92f53c92f807b9";
+    sha256 = "077hkvpmk3ghbqyb901w43b2m2a27lh8ddasyx1x7pdwyr2bjjl2";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    cp *.ttf $out/share/fonts/truetype
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Open Sans fonts";
+    longDescription = ''
+      Open Sans is a humanist sans serif typeface designed by Steve Matteson,
+      Type Director of Ascender Corp.
+    '';
+    homepage = https://en.wikipedia.org/wiki/Open_Sans;
```

Comment:
Why not `https://www.opensans.com/` ?

Reply:
Didn't see that there. I could also just download the fonts from their website also.

Reply:
I realized that shortly also. Also don't know if this should be called `opensans` or `open-sans`.
I'm :+1: on the second one because it's easier to read.

Reply:
Yheah I'd vote for `open-sans` too.